.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+---
+
```

Comment:
I think that the most important important thing for bugs is how to reproduce it.
The goal of these templates is to ask the minimum and effective amount of information. 

Maybe. instead of **Description** we can use **Quick Description** or **Short Description** can be a solution if you really want to add anything before the steps. otherwise the additional info can be used

what do you think?
@offtherailz @mbarto @tdipisa @allyoucanmap

Reply:
I think that the most important important thing for bugs is how to reproduce it.
The goal of these templates is to ask the minimum and effective amount of information. 
That's why we omitted Description, in favor of the How to reproduce.

Maybe. instead of **Description** we can use **Quick Description** or **Short Description** can be a solution if you really want to add anything before the steps. otherwise the additional info can be used

what do you think?
@offtherailz @mbarto @tdipisa @allyoucanmap

Reply:
I think that the most important important thing for bugs is how to reproduce it.
The goal of these templates is to ask the minimum and effective amount of information. 
That's why we omitted Description, in favor of the How to reproduce.

Maybe. instead of **Description** we can use **Quick Description** or **Short Description** can be a solution if you really want to add anything before the steps. otherwise the additional info can be used

what do you think?
@offtherailz @mbarto @tdipisa @allyoucanmap

Reply:
Too many discussions, a description is a description @MV88, please use Description.